rest/taskana-rest-spring/src/test/resources/taskana.properties

```diff
@@ -1,4 +1,4 @@
-taskana.roles.user=cn=ksc-users,cn=groups,OU=Test,O=TASKANA | teamlead-1 | teamlead-2 | user-1-1 | user-1-2 | user-2-1 | user-2-2 | user-b-1 | user-b-2
+taskana.roles.user=cn=ksc-users,cn=groups,OU=Test,O=TASKANA | teamlead-1 | teamlead-2 | user-1-1
```

Comment:
Is this the answer to my previous comment?

Reply:
exactely